docs/nf4_science/rnaseq/01_method.md

```diff
@@ -0,0 +1,507 @@
+Digest: sha256:4f00e7b2a09f3c8d8a9ce955120e177152fb1e56f63a2a6e186088b1250d9907
+Status: Downloaded newer image for community.wave.seqera.io/library/trim-galore:0.6.10--1bf8ca4e1967cd18
+community.wave.seqera.io/library/trim-galore:0.6.10--1bf8ca4e1967cd18
+```
+
+### 1.2. Spin up the container interactively
+
+```bash
+docker run -it -v ./data:/data community.wave.seqera.io/library/trim-galore:0.6.10--1bf8ca4e1967cd18
+```
+
+Your prompt will change to something like `(base) root@b645838b3314:/tmp#`, which indicates that you are now inside the container.
+
+The `-v ./data:/data` part of the command will enable us to access the contents of the `data/` directory from inside the container.
+
+```bash
+ls /data/reads
+```
+
+```console title="Output"
+ENCSR000COQ1_1.fastq.gz  ENCSR000COQ2_2.fastq.gz  ENCSR000COR2_1.fastq.gz  ENCSR000CPO1_2.fastq.gz
+ENCSR000COQ1_2.fastq.gz  ENCSR000COR1_1.fastq.gz  ENCSR000COR2_2.fastq.gz  ENCSR000CPO2_1.fastq.gz
+ENCSR000COQ2_1.fastq.gz  ENCSR000COR1_2.fastq.gz  ENCSR000CPO1_1.fastq.gz  ENCSR000CPO2_2.fastq.gzO
+```
+
+### 1.3. Run the first `fastqc` command
+
+Let's run `fastqc` to collect quality control metrics on the read data.
+
+```bash
+fastqc /data/reads/ENCSR000COQ1_1.fastq.gz
+```
+
+This should run very quickly:
+
+```console title="Output"
+application/gzip
+Started analysis of ENCSR000COQ1_1.fastq.gz
+Approx 5% complete for ENCSR000COQ1_1.fastq.gz
+Approx 10% complete for ENCSR000COQ1_1.fastq.gz
+Approx 15% complete for ENCSR000COQ1_1.fastq.gz
+Approx 20% complete for ENCSR000COQ1_1.fastq.gz
+Approx 25% complete for ENCSR000COQ1_1.fastq.gz
+Approx 30% complete for ENCSR000COQ1_1.fastq.gz
+Approx 35% complete for ENCSR000COQ1_1.fastq.gz
+Approx 40% complete for ENCSR000COQ1_1.fastq.gz
+Approx 45% complete for ENCSR000COQ1_1.fastq.gz
+Approx 50% complete for ENCSR000COQ1_1.fastq.gz
+Approx 55% complete for ENCSR000COQ1_1.fastq.gz
+Approx 60% complete for ENCSR000COQ1_1.fastq.gz
+Approx 65% complete for ENCSR000COQ1_1.fastq.gz
+Approx 70% complete for ENCSR000COQ1_1.fastq.gz
+Approx 75% complete for ENCSR000COQ1_1.fastq.gz
+Approx 80% complete for ENCSR000COQ1_1.fastq.gz
+Approx 85% complete for ENCSR000COQ1_1.fastq.gz
+Approx 90% complete for ENCSR000COQ1_1.fastq.gz
+Approx 95% complete for ENCSR000COQ1_1.fastq.gz
+Analysis complete for ENCSR000COQ1_1.fastq.gz
+```
+
+You can find the output files in the same directory as the original data:
+
+```bash
+ls /data/reads/ENCSR000COQ1_1_fastqc*
+```
+
+```console title="Output"
+/data/reads/ENCSR000COQ1_1_fastqc.html  /data/reads/ENCSR000COQ1_1_fastqc.zip
+```
+
+### 1.4. Trim adapter sequences with `trim_galore`
+
+Now let's run `trim_galore`, which bundles Cutadapt and FastQC, to trim the adapter sequences and collect post-trimming QC metrics.
+
+```bash
+trim_galore --fastqc /data/reads/ENCSR000COQ1_1.fastq.gz
+```
+
+The `--fastqc` flag causes the command to automatically run a QC collection step after trimming is complete.
+
+The output is very verbose:
```

Comment:
This is indeed very verbose. Might be be a good idea to abbreviate it here?

Reply:
Yeah I was thinking that too. Will do.

Reply:
done